src/components/ApplicationDetails/ApplicationDetails.tsx

```diff
@@ -82,14 +87,31 @@ const ApplicationDetails: React.FC<HacbsApplicationDetailsProps> = ({ applicatio
         data-test="application-details-test-id"
         headTitle={appDisplayName}
         breadcrumbs={[
-          { path: '/stonesoup/applications', name: 'Applications' },
+          <Badge key="badge" isRead>
```

Comment:
@rottencandy Any reason to switch from breadcrumb items to breadcrumbs only?

Reply:
The `breadcrumbitems` prop would only allow adding components to the end of breadcrumbs list, and the `breadcrumbs` prop did not support custom item components.

I added support for custom components to `breadcrumbs` so we could use it to render both paths and custom items at any position in the breadcrumbs list. And we wouldn't need `breadcrumbiems` anymore.